Dockerfile.desktop

```diff
@@ -55,16 +56,23 @@ COPY . /code
 
 WORKDIR /code
 
-# Install Firefox and Geckodriver
-RUN INSTALLER_DOWNLOAD_URL=https://raw.githubusercontent.com/hackebrot/install-firefox/master/install-firefox.sh \
-    && curl $INSTALLER_DOWNLOAD_URL -sSf | sh -s -- --firefox $FIREFOX_VERSION --geckodriver $GECKODRIVER_VERSION
-
-# Update to firefox nightly
-ARG FIREFOX_DOWNLOAD_URL=https://download.mozilla.org/?product=firefox-nightly-latest-ssl&lang=en-US&os=linux64
+# Download Firefox
+ARG FIREFOX_DOWNLOAD_URL=https://download.mozilla.org/?product=firefox${FIREFOX_VERSION}-latest-ssl&lang=en-US&os=linux64
```

Comment:
Ah, clever! 
IIUC, this is why you had the hyphen-prefixed `FIREFOX_VERSION` in the scripts above; `--build-arg="FIREFOX_VERSION=-nightly"`
So if no `FIREFOX_VERSION` is set it downloads Release channel, otherwise downloads `firefox-nightly-latest-ssl` or `firefox-beta-latest-ssl`. 👍